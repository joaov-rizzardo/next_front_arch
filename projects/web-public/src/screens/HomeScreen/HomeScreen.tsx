import { Text } from "@alura/design-system/components/Text";

export function HomeScreen() {
  return (
    <main>
      <Text tag="h1">Home</Text>
      <Text tag="p">Importando modulo local: @alura/utils/math/sum</Text>
    </main>
  );
}
