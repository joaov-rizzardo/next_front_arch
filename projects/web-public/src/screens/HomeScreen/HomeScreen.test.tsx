import { HomeScreen } from "./HomeScreen";
import { customRender } from "@alura/test-commons/react-testing-library";
const render = customRender((props) => {
  return <section>{props.children}</section>;
});

describe("HOmeScreen", () => {
  it("renders the page", () => {
    const { container } = render(<HomeScreen />);
    expect(container).toMatchSnapshot();
  });
});
