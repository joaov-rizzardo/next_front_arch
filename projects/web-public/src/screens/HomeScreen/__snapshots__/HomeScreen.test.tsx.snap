// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HOmeScreen renders the page 1`] = `
.c0 {
  font-family: "sans-serif";
}

<div>
  <main>
    <h1
      class="c0"
    >
      Home
    </h1>
    <p
      class="c0"
    >
      Importando modulo local: @alura/utils/math/sum
    </p>
  </main>
</div>
`;
